examples/pipelines/disk-uploader/example-vm-task.yaml

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
don't insert these tasks in this repo, instead create pipeline which will use resolver and reference task there with correct params

Reply:
Thanks!